Files/Views/Pages/Properties.xaml

```diff
@@ -5,25 +5,14 @@
     xmlns:Windows10version1903="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 8)"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:local="using:Files"
-    xmlns:local1="using:Files.Helpers"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
     d:DesignHeight="700"
     d:DesignWidth="400"
     KeyDown="Page_KeyDown"
     Loaded="Properties_Loaded"
-    RequestedTheme="{x:Bind local1:ThemeHelper.RootTheme}"
     Unloaded="Properties_Unloaded"
     mc:Ignorable="d">
-    <Page.Background>
```

Comment:
Why did you removed this?

Reply:
The `{x:Bind}` expressions below were causing `Wrong Thread` exceptions (probably because ApplicationViews runs in a separate UI thread). So I moved that login to `AppSettings_ThemeModeChanged`